bin/moveSequencingData.sh

```diff
@@ -133,76 +133,79 @@ SEQ_DIR="/groups/umcg-lab/${tmpDir}/sequencers/"
 
 log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "find ${SEQ_INCOMING_DIR}/ -mindepth 1 -maxdepth 1 -type d -o -type l"
 mapfile -t runs < <(find "${SEQ_INCOMING_DIR}/" -mindepth 1 -maxdepth 1 -type d -o -type l)
-
-for i in "${runs[@]}"
-do
+if [[ "${#runs[@]}" -eq '0' ]]
```

Comment:
added check when runs is empty or not.
It will "mess" up the indentation, no changes made (besides the lines i've commented)